entries/d/datadoghq.com.json

```diff
@@ -0,0 +1,13 @@
+{
+  "Datadog": {
+    "domain": "datadoghq.com",
+    "url": "https://app.datadoghq.com",
```

Comment:
```suggestion
```

Because going to `https://[domain]` works, the `url` is not necessary.

Reply:
There's no ability to sign in on the top-level domain, it redirects to `app.` so it seemed more appropriate to send folks there. But happy to remove it.